.env

```diff
@@ -2,4 +2,13 @@ POSTGRES_URL=jdbc:postgresql://apuopere-database:5432/apuope_db
 POSTGRES_USER=apuope_user
 POSTGRES_PASSWORD=apuope_password
 
-APP_URL_DEV=http://localhost:3000/
\ No newline at end of file
+APP_URL_DEV=http://localhost:3000/
+
+SPRING_DATASOURCE_URL=jdbc:postgresql://apuopere-database:5432/apuope_db
+SPRING_MAIL_HOST=smtp.gmail.com
+SPRING_MAIL_PORT=587
+SPRING_MAIL_USERNAME=mail.apuopere@gmail.com
+SPRING_MAIL_PASSWORD=tsnbgijhjtetbonu
+
+LLM_API_URL=https://aviary.fgl.rd.tuni.fi/v1/chat/completions
+LLM_API_KEY=sk-_yezpL9A8xUismP_X0Yu6A
```

Comment:
don't push env

Reply:
Shit this happens way too easily by accident. I thought that .gitignore would already prevent pushing .env. I'll look into blocking this.